universal-application-tool-0.0.1/conf/routes

```diff
@@ -10,10 +10,11 @@ GET     /count                      controllers.CountController.count
 GET     /message                    controllers.AsyncController.message
 
 # List all persons
-GET     /postgres                   controllers.PostgresController.list()
+GET     /postgres                   controllers.PostgresController.index()
 
 # Add a person
-GET     /postgres/add/:name         controllers.PostgresController.add(name: String)
+POST    /postgres/create            controllers.PostgresController.create(request: Request)
```

Comment:
Issue (blocking): this doesn't follow the routing convention in the README

Creating a new resource should be a POST to the resource name. This should be `POST /postgres`. The sync version can be `POST /postgresSync`

Reply:
Whoops. Fixed it.